src/worker/jobs/compute-stake-pool-events.ts

```diff
@@ -38,7 +38,7 @@ export async function computeStakePoolsEvents(_job: Job, extrinsicId?: string):
             for (const member of unbondingMembers) {
                 if (member.unbondingEras?.length) {
                     const unbondingComplete = member.unbondingEras.some(
-                        (unbondingEra) => currentEra[0].index >= unbondingEra.era
+                        (unbondingEra) => currentEra[0].index === unbondingEra.era
```

Comment:
```suggestion
                        (unbondingEra) => currentEra[0].index >= unbondingEra.era
```
why the === comparison? shouldn't be >= ?

Reply:
we want it to be checked only if its the same era